### PR TITLE
ontrack-enhance the chart

### PR DESCRIPTION
**Pull request by nicole5311**
# Description

including two figma page, technical diagram and angular jason.

### CODE REVIEW

File: docs/OnTrack/Data Analytics Platform/angular.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
+import { Component, VERSION } from '@angular/core';
```
Comment (lst97) [inline docs/OnTrack/Data Analytics Platform/angular.json:1]: This is a jsx file, this is not a valid json format. Also in this repo, we only need to upload documents not the code file. Please upload your code into your own repo and make a document here to link to your code repo. Thank you.

File: docs/OnTrack/Data Analytics Platform/Student progress overview.json [hunk 2]
```diff
@@ -0,0 +1 @@
...leAppPosition":null,"resourceDisplayName":null}},"version":"2.106.2","appStyles":"","createdAt":null,"appThemeId":null,"isFetching":false,"rootScreen":null,"isMobileApp":false,"testEntities":[],"isGlobalWidget":false,"customShortcuts":[],"mobileAppSettings":{"mobileOfflineModeEnabled":false},"customDocumentTitle":"","preloadedAppJSLinks":[],"markdownLinkBehavior":"never","multiScreenMobileApp":false,"queryStatusVisibility":true,"instrumentationEnabled":false,"pageLoadValueOverrides":[],"preloadedAppJavaScript":null,"urlFragmentDefinitions":[],"experimentalPerfFeatures":{"useRuntimeV2":false,"serverDepGraphEnabled":false},"responsiveLayoutDisabled":false,"inAppRetoolPillAppearance":"NO_OVERRIDE","loadingIndicatorsDisabled":false,"customDocumentTitleEnabled":false,"experimentalDataTabEnabled":false,"experimentalStabilityFeatures":{"lockModelForUpdates":false}}}},{"type":"WIDGET_REPOSITION2","payload":{"moves":[{"move":{},"moveType":"drag","widgetIds":["container2"]}],"largeScreen":true},"hideChangelogEntry":false}],"gitSha":null,"checksum":null,"createdAt":"2022-12-25T21:21:10.841Z","updatedAt":"2022-12-25T21:21:10.841Z","pageId":1691473,"userId":487238,"branchId":null},"modules":{}}
```
Comment (lst97) [inline docs/OnTrack/Data Analytics Platform/Student progress overview.json:1]: I think you will need to document the two code file shows how we can run your application, and what is this file for.

**Review by lst97 — CHANGES_REQUESTED** [review]
1. I can see your team submit different design and Figma, in this case, can you create a folder that use your name as parent folder name or use "design 1", "design 2" to separate you team design, so that it will not confuse future student. 

2. In the UML diagram, what is the diagram for? DB, Workflow or the Application, It is recommend that you should included a document to explain the diagram.

3. Same for the above comment, should include document about the diagram, shows why you design the diagram this way.

Thank you.

**Comment by maddernd** [comment]
This one is being cancelled as it fails all checks.

**Pull request closed without merging**